recipes/celiagg/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "13ff68c46e7a798bdd2af9f6a1c027dc9cac01e01e3845e11309ab02ecc32243" %}
+
+package:
+  name: celiagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
```

Comment:
Done. I couldn't find the documentation for this part of the `source` section, but I'm hoping this is a local rename that conda performs?